src/components/dropdown/dropdown.js

```diff
@@ -66,10 +66,18 @@ export default class Dropdown extends PureComponent {
 
   close = (evt) => {
     // IDK yet how to test using the ref in enzime
-    if (this.props.hoverable || (evt && this.domRef && this.domRef.current.contains(evt.target))) {
+    if (
+      this.props.hoverable
+      || (evt
+        && this.domRef
+        && this.domRef.current
```

Comment:
Just checking for this additional condition will lead to this.setState being called on an unmounted component.